src/core/hle/service/am/am.cpp

```diff
@@ -672,6 +677,30 @@ void ICommonStateGetter::GetCurrentFocusState(Kernel::HLERequestContext& ctx) {
     rb.Push(static_cast<u8>(FocusState::InFocus));
 }
 
+void ICommonStateGetter::IsVrModeEnabled(Kernel::HLERequestContext& ctx) {
+    LOG_WARNING(Service_AM, "(STUBBED) called");
+
+    IPC::ResponseBuilder rb{ctx, 3};
+    rb.Push(RESULT_SUCCESS);
+    rb.PushEnum<VrMode>(VrMode::Disabled);
```

Comment:
It should be sufficient to do:

```suggestion
    rb.PushEnum(VrMode::Disabled);
```

Since the member function deduces this from the give arguments.

Reply:
Fixed